### PR TITLE
Add send notification when invitation is created

### PR DESCRIPTION
**Pull request by CMaidana**
#### Trello ticket:

https://trello.com/c/1hNXuIRu/25-como-usuario-cuando-me-invitan-a-un-evento-debo-ser-notificado-2 

------
#### How I solved it:

Add send_notification function to app/models/invitation.rb

------
#### Refactors or changes not directly related to the main purpose of this PR:

------
#### Screenshots:

------

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -59,4 +61,14 @@ def public_fields_updated?
     name_changed? || address_changed? || address_changed? ||
       start_time_changed? || end_time_changed? || cancelled_changed?
   end
+
+  def send_notification
+    invitations do |invitation|
+      invitation.user.send_notification(
+         name,
+        'Event has change.',
```
Comment (DJA89) [inline app/models/event.rb:69]: This should be in Spanish

File: app/models/event.rb [hunk 2]
```diff
@@ -15,6 +15,8 @@ class Event < ApplicationRecord
 
   before_save :set_updated_event_at, if: :public_fields_updated?
 
+  after_update :send_notification
```
Comment (DJA89) [inline app/models/event.rb:18]: There's already another PR for this, please make only one Story per PR

File: app/models/invitation.rb [hunk 3]
```diff
@@ -3,8 +3,19 @@
...
   def new_updates_since_last_seen?
     changed_last_seen.present? && event.updated_event_at > changed_last_seen
   end
+
+  private
+
+  def send_notification
+    user.send_notification(
+      event.name,
+      'You have a new event invitation.',
```
Comment (DJA89) [inline app/models/invitation.rb:17]: This should be in Spanish

**Review by DJA89 — CHANGES_REQUESTED** [review]

File: app/models/event.rb (hunk 2 above)
Comment (CMaidana) [reply app/models/event.rb:18]: OK, I remove this feature

File: app/models/invitation.rb (hunk 3 above)
Comment (CMaidana) [reply app/models/invitation.rb:17]: OK

File: spec/requests/api/v1/invitations/update_spec.rb [hunk 4]
```diff
@@ -23,6 +23,10 @@
     }
   end
 
+  before(:each) do
```
Comment (DJA89) [inline spec/requests/api/v1/invitations/update_spec.rb:26]: Tenés que hacer tests de la notificación, fijate los de la communication

**Review by DJA89 — APPROVED** [review]

**Review by guzmanmo — APPROVED** [review]

**Pull request merged**